tests/visual-regression/specs/visual-snapshots.test.js

```diff
@@ -0,0 +1,189 @@
+import { visitAdminPage } from '@wordpress/e2e-test-utils';
+
+const screenshotOptions = {
+	fullPage: true,
+};
+
+async function hideDynamicElements( elements ) {
+	for ( let i = 0; i < elements.length; i++ ) {
+		const elementOnPage = await page.$( elements[ i ] );
+		if ( elementOnPage ) {
+			await page.$eval( elements[ i ], ( el ) => {
```

Comment:
I think you can also do `elementOnPage.evaluate` and the callback will receive the right DOM element:
https://github.com/puppeteer/puppeteer/blob/v5.5.0/docs/api.md#elementhandleevaluatepagefunction-args